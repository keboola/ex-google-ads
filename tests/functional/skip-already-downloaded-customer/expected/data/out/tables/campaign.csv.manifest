{
  "primary_key": [
    "customerId",
    "id"
  ],
  "columns": [
    "customerId",
    "id",
    "name",
    "status",
    "servingStatus",
    "adServingOptimizationStatus",
    "advertisingChannelType",
    "startDate",
    "endDate"
  ],
  "incremental": true
}