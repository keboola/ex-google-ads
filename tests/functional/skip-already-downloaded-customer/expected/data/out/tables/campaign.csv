"3260444947","1015595176","test","ENABLED","SERVING","OPTIMIZE","SEARCH","2017-12-11","2037-12-30"
"3260444947","6621804912","Brand awareness and reach-Display-1","ENABLED","SERVING","OPTIMIZE","DISPLAY","2019-10-01","2037-12-30"
"9250499593","15243152140","618e49b68ce4a","ENABLED","SERVING","OPTIMIZE","SEARCH","2021-11-12","2037-12-30"
